unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/SceneBoundariesController/Tests/SBC_Asserts.cs

```diff
@@ -20,7 +22,7 @@ namespace SceneBoundariesCheckerTests
 {
     public static class SBC_Asserts
     {
-        public static IEnumerator EntitiesAreBeingCorrectlyRegistered(ParcelScene scene)
+        public static async Task EntitiesAreBeingCorrectlyRegistered(ParcelScene scene)
```

Comment:
Is it possible to pass cToken and handle cancelations for all these tasks (maybe at least in some Dispose?)

Reply:
All those tasks in SBC_Asserts are being used by 2 different test fixtures, we don't handle cancellations in tests at least it explicitly required